### PR TITLE
Add Charge api

### PR DESCRIPTION
**Pull request by tomas-zijdemans-vipps**

### CODE REVIEW

File: sample_code/charge_sample.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+const charge = await client.charge.create(token, agreementId, {
+  amount: 2500,
+  description: "MyNews Digital",
+  orderId: crypto.randomUUID(),
+  due: tenDaysFromToday,
+  retryDays: 5,
+  transactionType: "DIRECT_CAPTURE",
+});
+
+// Check if the charge was create successfully
```
Comment (flinkgutt) [inline sample_code/charge_sample.ts:89]:
```suggestion
// Check if the charge was created successfully
```

File: sample_code/charge_sample.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  console.error("😟 Error creating charge 😟");
+  console.error(charge.error);
+  Deno.exit(1);
+}
+
+const chargeId = charge.data.chargeId;
+
+const chargeInfo = await client.charge.info(token, agreementId, chargeId);
+
+// Check if the charge was create successfully
```
Comment (flinkgutt) [inline sample_code/charge_sample.ts:100]:
```suggestion
// Check if the charge info was fetched successfully
```

File: src/apis/types/charge_types.ts [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+  RecurringCurrencyV3,
+  RecurringTransactionType,
+} from "./recurring_types.ts";
+
+export type CreateChargeV3 = {
+  /**
+   * Amount to be paid by the customer.
+   *
+   * Amounts are specified in minor units.
+   * For Norwegian kroner (NOK) that means 1 kr = 100 øre. Example: 499 kr = 49900 øre.
```
Comment (flinkgutt) [inline src/apis/types/charge_types.ts:12]:
```suggestion
   * For Norwegian kroner (NOK) that means 1 kr = 100 øre/cents. Example: 499 kr = 49900 øre/cents.
```

**Review by flinkgutt — CHANGES_REQUESTED** [review]
Just look over the suggested comment changes and let me know the end result. Everything else looks good as far as I can see.

File: sample_code/charge_sample.ts (hunk 1 above)
Comment (tomas-zijdemans-vipps) [reply sample_code/charge_sample.ts:89]: Thanks!

**Pull request merged**